.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    with:
+      posargs: --openmp-expected=True
+      coverage: codecov
+      envs: |
+        # Code style
+        - linux: style
+        # Docs
+        - linux: build_docs
+        # Standard tests
+        - linux: py36-test-oldestdeps
+        - linux: py37-test
+        - linux: py38-test
+        - linux: py38-test-dev
+        - macos: py38-test-dev
+          posargs: --openmp-expected=False
+        - windows: py36-test
+        - windows: py38-test-dev
+        # Test with more compilers, for the OpenMP helpers
+        - macos: py38-test-osxclang-conda
+        - linux: py38-test-linuxgcc-conda
+        # Test downstream packages
+        - linux: py39-downstream
+      env:
+        OA_WINDOWS_RUNNER: windows-2019
```

Comment:
@ConorMacBride - I can't figure out what the correct level of indentation needs to be, nothing seems to work. Do you have any ideas?

Reply:
Doesn't the web editor tell you if you use GitHub web UI to edit the file? You indent/unindent until the red swiggles go away... 😆

Reply:
Apparently you can't pass custom environment variables to a reusable workflow: https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations

I'm working on a new `runner` input.

Reply:
You can now remove the `OA_WINDOWS_RUNNER` and do:
```yaml
- windows: py36-test
  runs-on: windows-2019
- windows: py38-test-dev
  runs-on: windows-2019
```

Reply:
Thanks! I've added that here.